Documentation UI changes for the overhaul

lfrancke opened the pull request.

mojavelinux committed:
replace default branch name with HEAD

mojavelinux committed:
set docs version to ~ to indicate it is versionless

mojavelinux committed:
update terminology for referring to the default branch

mojavelinux committed:
revert value of version key since docs are still built using Antora 2

mojavelinux committed:
document the page.author property in the UI model

mojavelinux committed:
set docs version to ~

Commit:
Added font dependency / Changed font to Titillium Web

Commit:
Color adjustments

Commit:
Further color adjustments

Commit:
custom bundle yaml

Commit:
fixed lint errors

Commit:
logo and favicon

Commit:
stackable logo + doc link in header

Commit:
sub navbar

Commit:
contact button

Commit:
hide explore panel

Commit:
expandable sub nav

Commit:
fixed lint errors

mojavelinux committed:
assume value of toctitle page attribute is already escaped

mojavelinux committed:
add search input behind a flag

- revise CSS to accomodate search input
- show search input if SITE_SEARCH_ENABLED env var is set
- remove Resources item in navbar

mojavelinux committed:
merge !153

add search input behind a flag

mojavelinux committed:
look for env.SITE_SEARCH_PROVIDER instead of env.SITE_SEARCH_ENABLED

mojavelinux committed:
include search-scripts partial if SITE_SEARCH_PROVIDER env var is set

Commit:
Lunr search input field integration

Commit:
Lunr search css integration

Commit:
Lunr search css adjusments

mojavelinux committed:
resolves #181 prevent overscroll event in scrollable region from bubbling

mojavelinux committed:
merge !154

resolves #181 prevent overscroll event in scrollable region from bubbling

mojavelinux committed:
clarify that canonical URL is only set if site.url is an absolute URL

mojavelinux committed:
change template-variables-table ID to template-variables-ref

mojavelinux committed:
apply font weight bold to h4

Commit:
dropdown menu

Commit:
fixed lint errors

mojavelinux committed:
document the site.homeUrl property in the UI model; clarify how it differs from site.url and siteRootPath

mojavelinux committed:
drop check for outdated siteRootUrl property; document using siteRootPath as fallback for site.url

mojavelinux committed:
assume special chars in description and keywords are already escaped; detag description

mojavelinux committed:
upgrade UI preview to Asciidoctor.js 2

mojavelinux committed:
fix clone command on set up page so it works with copy to clipboard

Commit:
custom landing page

Commit:
custom landing page

ggrossetie committed:
Style the search bar when disabled

mojavelinux committed:
update dependency lock file

mojavelinux committed:
upgrade gulp-stylelint so dependencies can be installed on Node.js 16

mojavelinux committed:
merge !155

Style the search box when disabled

Commit:
re-enabled pushing ui bundle to git

lfrancke committed:
Update content

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/stackabletech/documentation-ui?pullRequest=2) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/stackabletech/documentation-ui?pullRequest=2) before we can accept your contribution.<br/>**0** out of **4** committers have signed the CLA.<br/><br/>:x: mojavelinux<br/>:x: dropclick<br/>:x: Mogztter<br/>:x: edward-encoord<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/stackabletech/documentation-ui?pullRequest=2) it.</sub>

edward-encoord committed:
Same format of "videoblack" as "imageblock" (#184)

mojavelinux committed:
add vertical-align middle to video iframe; remove margin on video title; add video to preview

mojavelinux committed:
merge !157

resolves #184 use same format of "videoblock" as "imageblock"

lfrancke committed:
Merge remote-tracking branch 'upstream/master' into sven-wip

# Conflicts:
#	package-lock.json
#	package.json
#	preview-src/index.adoc
#	src/css/header.css
#	src/partials/header-content.hbs

lfrancke committed:
Resolve merge conflicts

lfrancke committed:
Various updates

Change URL of published bundle so it becomes clearer in the repo
Fix link
Fix a few dependencies
